.github/workflows/planemo.yml

```diff
@@ -27,7 +40,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7']
+        python-version: ['3.10']
```

Comment:
we could test multiple versions if we like.
Historically we tested for the lower bound. But we could test for lower and upper-supported python versions.

3.8 and 3.11?

Reply:
I've included 3.7 & 3.11, though I'm not sure how relevant it is at this stage (planemo_combine_outputs): The tests have already ran at that time, right ?